rust/src/services/messages/state_machine.rs

```diff
@@ -23,9 +23,11 @@ use crate::{
     PetError,
 };
 
-/// The request type handled by [`StateMachineService`]
+/// [`StateMachineService`] request type
 #[derive(Debug, From)]
 pub struct StateMachineRequest(MessageOwned);
+
+/// `[StateMachineService`] response type
 pub type StateMachineResponse = Result<(), StateMachineError>;
 
```

Comment:
the `StateMachineService` struct and its methods should be documented.

Reply:
Done. I documented only the public parts though.